infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -246,8 +361,14 @@ cloud_run_config = {
       }
     ]
   }
+  // GCP us-west2 is California, USA
   us-west2 = {
     environment_variables = [
+      {
+        name = "DEFAULT_AWS_BASE_URL",
+        // AWS us-west-1 is California, USA
+        value = "https://prod-registry-k8s-io-us-west-1.s3.dualstack.us-west-2.amazonaws.com",
```

Comment:
Typo: us-west-1 and us-west-2 in same URL, should match I think

Reply:
fixed!

Reply:
thanks Ben